virga/plugins/graphql/route.py

```diff
@@ -79,12 +69,10 @@ async def _handle_request(self, request: Request) -> Response:
         if result.errors:
             response["errors"] = [format_error(e) for e in result.errors]
 
-        status_code = (
-            status.HTTP_400_BAD_REQUEST if result.errors else status.HTTP_200_OK
+        return ORJSONResponse(
```

Comment:
nice! I forgot graphql never raises errs

Reply:
did you need a test for this? some fake graphql schema that raises an error - to demo the always-200 response and what `errors` will contain?